python/setup.py

```diff
@@ -38,7 +38,6 @@
     "python-dateutil;python_version<'3.7'",
     # Python 3.7+
     "attrs >= 20.1.0;python_version>='3.7'",
-    "cattrs == 1.1.2;python_version>='3.7'",
```

Comment:
oops, actually we need to still specify a cattrs dep here ([ci failed](https://github.com/looker-open-source/sdk-codegen/runs/4902006917?check_suite_focus=true)).

```
cattrs >= 1.3;python_version>='3.7'
```

should have the fix?

Reply:
Done, also updated `Pipfile` with `cattrs = ">=1.3"`.